lib/querystring.js

```diff
@@ -212,12 +212,12 @@ QueryString.parse = QueryString.decode = function(qs, sep, eq, options) {
   sep = sep || '&';
   eq = eq || '=';
 
-  const obj = {};
-
   if (typeof qs !== 'string' || qs.length === 0) {
-    return obj;
+    return {};
   }
 
+  const obj = Object.create(null);
```

Comment:
This is very slow compared to `var x = {}` (last we checked, like 80+ times slower).